espaloma/graphs/tests/test_deploy.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import espaloma as esp
+
+def test_methane():
+    ff = esp.graphs.legacy_force_field.LegacyForceField("smirnoff99Frosst")
+    g = esp.Graph('CCCC')
```

Comment:
mismatch between smiles and test name

Reply:
Ops

Reply:
Resolved in https://github.com/choderalab/espaloma/pull/39/commits/75e8919be84520f2fccbcb83d421ed1ebaff8757